assert/assertions.go

```diff
@@ -811,6 +811,27 @@ func Panics(t TestingT, f PanicTestFunc, msgAndArgs ...interface{}) bool {
 	return true
 }
 
+// PanicsWithValue asserts that the code inside the specified PanicTestFunc panics, and that
+// the recovered panic value equals the expected panic value.
+//
+//   assert.PanicsWithValue(t, "crazy error", func(){
+//     GoCrazy()
+//   }, "Calling GoCrazy() should panic with 'crazy error'")
```

Comment:
Can you please remove the message? A example error message is added automatically to generated `PanicWithValuef` functions